lib/screens/play/scrying_page.dart

```diff
@@ -41,20 +41,22 @@ class ScryingPage extends StatelessWidget {
 
   Widget _getScryButton(
       BuildContext context, Model.Card card, ScryOption scryOption) {
-    return Container(
-        padding: EdgeInsets.symmetric(vertical: 5),
-        width: MediaQuery.of(context).size.width * 0.7,
-        child: RaisedButton(
-          shape: RoundedRectangleBorder(
-            side: BorderSide(color: Colors.black, width: 1),
-            borderRadius: BorderRadius.circular(10),
-          ),
-          onPressed: () {
-            state.replaceScryedCard(card, scryOption);
-          },
-          color: scryOption.getColour(),
-          child:
-              Text(scryOption.getDescription(), style: TextStyle(fontSize: 30)),
-        ));
+    return Visibility(
+        visible: state.showScryButtons,
```

Comment:
If we go with my suggested state tweak, we'll need another strategy here. I'd suggest we revert this back to the simple container, and extract the relevant parts to reuse from a new component (`view_discard_page` ?).

So we'd end up with a generic `CardViewer` or some such, which you'd pass a list of Cards and a transformer to translate a card to a widget. The view discard page would pass the discard list, and a simple transformer that just renders the card. The scrying page would pass the scry list, and a transformer that includes the scry buttons.

Then we'll want a switch in the game screen or wherever, to pay attention to the `viewing_discard` bool and show the view discard page appropriately.

It's a bit more code, but it keeps things explicit and separates our responsibilities nicely. Another advantage of separating out a `view_discard_page` is that we can make that one have a 'close' button, which we'd hook into a state method so you can return to the actual game. We wouldn't want this on the scrying page, which would require more boolean magic to toggle on/off with the current approach.